x-pack/filebeat/tests/integration/shipper_test.go

```diff
@@ -185,6 +212,61 @@ processors:
 		require.NoError(t, err, "error doing search request: %s", err)
 		return res.Hits.Total.Value == 2
 	}, 30*time.Second, 250*time.Millisecond, "never found 2 documents")
-	// ToDo add comparison of docs to make sure they are the same
-	// for example right now input.type is being overwritten with shipper
+
+	res, err := es.Search().
```

Comment:
This error here is not checked, which result in the test failing without a clear reason.

Reply:
thanks, fixed